### ISSUE TITLE
Deprecate / Delete Repo

### ISSUE THREAD

**Issue by dwt** [zopefoundation/cmf.zmiicons#1]
Hi there,

I propose to remove this repository, as it's sole purpose is to load a one icon iconfont, that is then used in Products.CMFCore to provide the icon for the cookie-crumbler object.

Since https://github.com/zopefoundation/cmf.zmiicons now provides the full FontAwesome 5, this font is no longer needed (instead all objects are encouraged to choose a FontAwesome icon if possible).

The dependency on this repo has already been removed.

@icemac Do you agree? Do I / we need buy-in from additional people?

**Comment by mgedmin** [comment]
> Since https://github.com/zopefoundation/cmf.zmiicons now provides the full FontAwesome 5, this repo and it's contained font is no longer needed

Um, https://github.com/zopefoundation/cmf.zmiicons _is_ "this repo".  Did you accidentally paste the wrong URL?

**Comment by rbu** [comment]
I think @dwt meant to say that https://github.com/zopefoundation/zmi.styles/ now contains full FontAwesome

**Comment by icemac** [comment]
@dwt Agreed. This is the last comment in this Repos before its deletion.
